examples/java/src/main/java/org/apache/beam/examples/complete/TfIdf.java

```diff
@@ -121,7 +122,8 @@
     Set<URI> uris = new HashSet<>();
     if (absoluteUri.getScheme().equals("file")) {
       File directory = new File(absoluteUri);
-      for (String entry : directory.list()) {
+      Optional<String[]> optional = Optional.fromNullable(directory.list());
```

Comment:
minor point: probably don't need to put it in a variable that is used just once on the next line